client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -55,6 +56,14 @@ const PatternAssembler: Step = ( { navigation, flow } ) => {
 		activePosition,
 	};
 
+	const siteInfo = useMemo(
```

Comment:
Curious, why do we need `useMemo` here? There is no heavy computation at all

Reply:
site?.title is always `undefined`, tested on default site info & customized info
![image](https://user-images.githubusercontent.com/10071857/211000410-870a7de3-f2f7-4e8e-9e97-e72f5e69e9fb.png)
Patter 5601-174455321 should have both title & tagline

Reply:
Oops...I use the wrong variable 🤦‍♂️

Reply:
Fixed the site title issue and removed the `useMemo` 😉

Reply:
Work great